at_end2end_test/test/sharing_key_test.dart

```diff
@@ -51,20 +51,22 @@ void main() {
   /// 3. Get method - lookup verb
   test('Share a key to sharedWith atSign and lookup from sharedWith atSign',
       () async {
-    var lastNumber = Random().nextInt(50);
-    var phoneKey = AtKey()
-      ..key = 'phone_$lastNumber'
+    var uuid = Uuid();
+    // Generate  uuid
+    var randomValue = uuid.v4();
+    var phoneNumberKey = AtKey()
+      ..key = 'phoneNumber$randomValue'
       ..sharedWith = sharedWithAtSign
       ..metadata = (Metadata()..ttl = 120000);
 
     // Appending a random number as a last number to generate a new phone number
     // for each run.
-    var value = '+1 100 200 30$lastNumber';
+    var value = '+91 901920192';
     // Setting currentAtSign atClient instance to context.
     await AtClientManager.getInstance().setCurrentAtSign(
         currentAtSign, namespace, TestUtils.getPreference(currentAtSign));
     var putResult =
-        await currentAtSignClientManager?.atClient.put(phoneKey, value);
+        await currentAtSignClientManager?.atClient.put(phoneNumberKey, value);
     expect(putResult, true);
     var isSyncInProgress = true;
     currentAtSignClientManager?.syncService.sync(onDone: (syncResult) {
```

Comment:
A call to sync is not required for this test. Probably not harmful but not required.

Reply:
The reason I mention this is that app devs are not expected to call sync at all. Sync should happen irrespectively.